<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditStopDialog</class>
 <widget class="QDialog" name="EditStopDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mainTab">
      <attribute name="title">
       <string>Stop</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <item row="1" column="0">
        <widget class="QGroupBox" name="platfBox">
         <property name="title">
          <string>Platform</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QRadioButton" name="platfRadio">
            <property name="text">
             <string>Main</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QRadioButton" name="depotRadio">
            <property name="text">
             <string>Depot</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="mainPlatfSpin">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="depotSpin">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" rowspan="2">
        <widget class="QGroupBox" name="descrBox">
         <property name="title">
          <string>Description</string>
         </property>
         <layout class="QVBoxLayout" name="descriptionLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QPlainTextEdit" name="descriptionEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="mainBox">
         <property name="title">
          <string>Time</string>
         </property>
         <layout class="QGridLayout" name="stopBoxLayout">
          <item row="1" column="1">
           <widget class="QTimeEdit" name="arrivalTimeEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Arrival</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Station</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Departure</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTimeEdit" name="departureTimeEdit"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="timeTab">
      <attribute name="title">
       <string>Time</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Arrival</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="prevStEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Previous Station</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Line Max.Speed</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Speed</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Train Max.Speed</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Departure</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QTimeEdit" name="calcTimeArr"/>
       </item>
       <item row="5" column="2">
        <widget class="QTimeEdit" name="calcTimeDep"/>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="calcTimeBut">
         <property name="text">
          <string>Re-Calc</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="applyTimeBut">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="speedEdit">
         <property name="suffix">
          <string> Km/h</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="lineSpeedBox">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="suffix">
          <string> Km/h</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="trainSpeedBox">
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="showGroupSeparator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> Km/h</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Line</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="currentLineText">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rsTab">
      <attribute name="title">
       <string>Rollingstock</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="coupledBox">
         <property name="title">
          <string>Coupled</string>
         </property>
         <layout class="QVBoxLayout" name="coupledLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QListView" name="coupledView"/>
          </item>
          <item>
           <widget class="QPushButton" name="editCoupledBut">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="uncoupledBox">
         <property name="title">
          <string>Uncoupled</string>
         </property>
         <layout class="QVBoxLayout" name="uncoupledLayout">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QListView" name="uncoupledView"/>
          </item>
          <item>
           <widget class="QPushButton" name="editUncoupledBut">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="rollingstockBox">
         <property name="title">
          <string>Job Asset</string>
         </property>
         <layout class="QGridLayout" name="trainAssetGridLayout">
          <item row="0" column="1">
           <widget class="QLabel" name="assetBeforeLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Asset Before Stop</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QListView" name="assetBeforeView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QListView" name="assetAfterView">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="assetAfterLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Asset After Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="infoTab">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>641</width>
            <height>584</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="infoLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Infos here...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="passingsBox">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="title">
              <string>Passings / Crossings</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QPushButton" name="calcPassingsBut">
                <property name="text">
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Passings</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="passingsView"/>
              </item>
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Crossings</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableView" name="crossingsView"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
